spring-integration-mongodb/src/main/java/org/springframework/integration/mongodb/metadata/package-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * 
+ */
+/**
+ * @author SenthilArumugam SP
```

Comment:
`package-info.java` must not contains an author name, but JavaDocs is required.
As well as new line in the end of each file.